docs/pages/config/config_files.md

```diff
@@ -32,28 +32,28 @@ for this file in one of these locations, in this order:
 1. In the default database location, if the database directory isn't specified
 1. In the location specified with '--lrl /path/to/dbname.lrl` on command line
 
-Lines begining with '#' are treated as comments and skipped.  Other lines should consist of a directive and its value.
+Lines beginning with '#' are treated as comments and skipped.  Other lines should consist of a directive and its value.
 
 ## Database tunables.
 
 ### Cache size
 
-Cache size can be set with the the 'cache' tunable. It takes a cache size and a unit (kb, mb, gb).  For example
+Cache size can be set with the the 'cache' tunable. It takes a cache size and a unit (KB, MB, GB).  For example
```

Comment:
We only accept lower case version of those.  Should probably fix that, but for now at least going to make the docs match the code.